.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,33 +0,0 @@
----
-name: Bug report
-about: Create a report to help us improve
-title: ''
-labels: bug
-assignees: ''
-
----
-
-**Describe the bug**
-A clear and concise description of what the bug is.
-
-**To Reproduce**
-Steps to reproduce the behavior:
-
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
-
-**Expected behavior**
-A clear and concise description of what you expected to happen.
-
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
-
-**Versions (please complete the following information):**
-
-- OS: [e.g. `Windows 10`, `macOS Catalina`, `Ubuntu 16.04`]
-- `santa` CLI: [e.g. `0.0.1`]
```

Comment:
☝️ These examples

Reply:
Yeah will restore that for now.

Can you spec an issue for a CLI command that will allow santa users to quickly print diagnostics that they can copy over to the issue. E.g.:

```
$ santa diagnostics
```
```
$ santa diagnostics --md
```

It should even try to use e.g. `pbcopy` under the hood too.

TBD but we can probably already agree that such a command will greatly the project.